packages/postgres/tsconfig.json

```diff
@@ -14,7 +14,11 @@
     "esModuleInterop": true,
     "outDir": "./dist/cjs",
     "declaration": true,
-    "composite": true
+    "composite": true,
+    "types": [
+      "pg",
+      "sqlstring"
```

Comment:
why is it needed with this change to list all external types in this tsconfig:types section?

Reply:
Packages' tsconfig.json files were updated to have compilerOptions.types correctly set, preventing redundant TS compilation from checking types of every node_modules/@types/* package (even not used in a particular monorepo package), which would occur every time a monorepo package is being built. This is because of having a single common node_modules shared across monorepo packages.